lib/matplotlib/axes/_axes.py

```diff
@@ -6457,7 +6457,21 @@ def hist2d(self, x, y, bins=10, range=None, normed=False, weights=None,
 
         Other parameters
         ----------------
-        kwargs : :meth:`pcolorfast` properties.
+        cmap : [None, Colormap, string], optional
```

Comment:
Would it be okay to get rid of the square brackets? I don't think they're anywhere else (or at least haven't been in recent doc PRs)